pinc/ProjectState.inc

```diff
@@ -272,22 +277,17 @@ function construct_star_metal_sql()
 
 // -----------------------------------------------------------------------------
 // these functions can be removed later
```

Comment:
This comments apply to the 4 functions following.

I only removed `get_forum_id_for_project_state` to prevent propagating the odd return type (and keep this PR somewhat contained as some of the wrappers have a large amount of usage).